Formula/apng2gif.rb

```diff
@@ -0,0 +1,20 @@
+class Apng2gif < Formula
+  desc "Convert APNG animations into animated GIF format."
+  homepage "https://apng2gif.sourceforge.io/"
+  url "https://downloads.sourceforge.net/apng2gif/apng2gif-1.8-src.zip"
+  sha256 "9a07e386017dc696573cd7bc7b46b2575c06da0bc68c3c4f1c24a4b39cdedd4d"
+
+  depends_on "libpng"
+
+  def install
+    inreplace "Makefile", "gcc", ENV.cc
+    inreplace "Makefile", "-s", ""
```

Comment:
This will need to be reported and fixed upstream in that case, as we don't want to be patching the Makefile of a brand new formula.